### PR TITLE
OCPBUGS#9364: Create bound SA tokens externally

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.11+

Issue: https://issues.redhat.com/browse/OCPBUGS-9364

Link to docs preview: https://57135--docspreview.netlify.app/openshift-enterprise/latest/authentication/bound-service-account-tokens.html#bound-sa-tokens-configuring_bound-service-account-tokens

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/bound-sa-tokens-configuring-externally.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+* You have created a service account. This procedure assumes that the service account is named `build-robot`.
+
+.Procedure
+
+* Create the bound service account token outside the pod by running the following command:
++
+[source,terminal]
+----
+$ oc create token <service_account>
+----
```
Comment (gkarager) [inline modules/bound-sa-tokens-configuring-externally.adoc:21]: @xenolinux It would be helpful if an example is added here.

Comment (xenolinux) [reply modules/bound-sa-tokens-configuring-externally.adoc:21]: @gkarager Would you be able to help me with the service token example?

**Comment by xingxingxia** [comment]
@zxiao-redhat could you kindly help review? Thanks!

File: modules/bound-sa-tokens-configuring-externally.adoc (hunk 1 above)
Comment (zxiao-redhat) [reply modules/bound-sa-tokens-configuring-externally.adoc:21]:
```suggestion
----

Example:
$ oc create serviceaccount test-sa 
> serviceaccount/test-sa created

$ oc create token test-sa         
> eyJhbGciOiJSUzI1NiIsImtpZCI6IkY2M1N4MHRvc2xFNnFSQlA4eG9GYzVPdnN3NkhIV0tRWmFrUDRNcWx4S0kifQ.eyJhdWQiOlsiaHR0cHM6Ly9pc3N1ZXIyLnRlc3QuY29tIiwiaHR0cHM6Ly9pc3N1ZXIxLnRlc3QuY29tIiwiaHR0cHM6Ly9rdWJlcm5ldGVzLmRlZmF1bHQuc3ZjIl0sImV4cCI6MTY3OTU0MzgzMCwiaWF0IjoxNjc5NTQwMjMwLCJpc3MiOiJodHRwczovL2lzc3VlcjIudGVzdC5jb20iLCJrdWJlcm5ldGVzLmlvIjp7Im5hbWVzcGFjZSI6ImRlZmF1bHQiLCJzZXJ2aWNlYWNjb3VudCI6eyJuYW1lIjoidGVzdC1zYSIsInVpZCI6ImM3ZjA4MjkwLWIzOTUtNGM4NC04NjI4LTMzMTM1NTVhNWY1OSJ9fSwibmJmIjoxNjc5NTQwMjMwLCJzdWIiOiJzeXN0ZW06c2VydmljZWFjY291bnQ6ZGVmYXVsdDp0ZXN0LXNhIn0.WyAOPvh1BFMUl3LNhBCrQeaB5wSynbnCfojWuNNPSilT4YvFnKibxwREwmzHpV4LO1xOFZHSi6bXBOmG_o-m0XNDYL3FrGHd65mymiFyluztxa2lgHVxjw5reIV5ZLgNSol3Y8bJqQqmNg3rtQQWRML2kpJBXdDHNww0E5XOypmffYkfkadli8lN5QQD-MhsCbiAF8waCYs8bj6V6Y7uUKTcxee8sCjiRMVtXKjQtooERKm-CH_p57wxCljIBeM89VdaR51NJGued4hVV5lxvVrYZFu89lBEAq4oyQN_d6N1vBWGXQMyoihnt_fQjn-NfnlJWk-3NSZDIluDJAv7e-MTEk3geDrHVQKNEzDei2-Un64hSzb-n1g1M0Vn0885wQBQAePC9UlZm8YZlMNk1tq6wIUKQTMv3HPfi5HtBRqVc2eVs0EfMX4-x-PHhPCasJ6qLJWyj6DvyQ08dP4DW_TWZVGvKlmId0hzwpg59TTcLR0iCklSEJgAVEEd13Aa_M0-faD11L3MhUGxw0qxgOsPczdXUsolSISbefs7OKymzFSIkTAn9sDQ8PHMOsuyxsK8vzfrR-E0z7MAeguZ2kaIY7cZqbN6WFy0caWgx46hrKem9vCKALefElRYbCg3hcBmowBcRTOqaFHLNnHghhU1LaRpoFzH7OUarqX9SGQ
```

Comment (zxiao-redhat) [reply modules/bound-sa-tokens-configuring-externally.adoc:21]: The token is clean. You can decode the token here: https://jwt.io

Comment (xenolinux) [reply modules/bound-sa-tokens-configuring-externally.adoc:27]: Hello @zxiao-redhat Thank you!
I added the example output for the `oc create token` command.

**Review by kowen-rh — APPROVED** [review]
LGTM, thank you! :+1:

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

**Pull request merged**